.goreleaser.yaml

```diff
@@ -1,39 +1,213 @@
+    
+    # Dependencies for the HTTP server
+    dependencies:
+      - adduser
+      - lsb-base (>= 3.0-6)
+    
+    contents:
+      # GoReleaser automatically includes binaries from the referenced build
+      
+      # Systemd service file
+      - src: ./packaging/go-template-httpserver.service
+        dst: /lib/systemd/system/go-template-httpserver.service
+        type: config
+        file_info:
+          mode: 0644
+          mtime: "{{ .CommitDate }}"
+      
+      # Default configuration
+      - src: ./packaging/go-template-httpserver.default
+        dst: /etc/default/go-template-httpserver
+        type: config
+        file_info:
+          mode: 0644
+          mtime: "{{ .CommitDate }}"
+      
+      # Create log directory with correct ownership
+      - dst: /var/log/go-template
+        type: dir
+        file_info:
+          mode: 0755
+          owner: go-template
+          group: go-template
+    
+    scripts:
+      postinstall: ./packaging/postinstall.sh
+      preremove: ./packaging/preremove.sh
+      postremove: ./packaging/postremove.sh
+    
+    # Set consistent modification time for reproducibility
+    mtime: "{{ .CommitDate }}"
+
+  - id: cli
+    package_name: go-template-cli
+    file_name_template: >-
+      {{ .PackageName }}_{{ .Version }}_{{ .Os }}_
+      {{- .Arch }}
+      {{- if not (eq .Amd64 "v1") }}{{ .Amd64 }}{{ end }}
+    
+    ids:
+      - cli
+    
+    vendor: Flashbots
+    homepage: https://github.com/flashbots/go-template
+    maintainer: Flashbots <info@flashbots.net>
+    description: |
+      Go Template CLI
+      A command-line interface for the Go template project.
+    license: MIT
+    
+    formats:
+      - deb
+      - rpm
+    
+    version_metadata: ""
+    umask: 0o022
+    bindir: /usr/bin
+    
+    # Set consistent modification time for reproducibility
+    mtime: "{{ .CommitDate }}"
+
+# Reproducible source archives
+source:
+  enabled: true
+  name_template: "{{ .ProjectName }}-{{ .Version }}"
+  format: tar.gz
+
+# Checksums for package verification
+checksum:
+  name_template: "checksums.txt"
+  algorithm: sha256
+
+# Reproducible changelog
+changelog:
+  use: github
+  sort: asc
+  filters:
+    exclude:
+      - "^docs:"
+      - "^test:"
+      - "merge conflict"
+      - Merge pull request
+      - Merge remote-tracking branch
+      - Merge branch
+  groups:
+    - title: Features
+      regexp: "^.*feat[(\\w)]*:+.*$"
+      order: 0
+    - title: "Bug fixes"
+      regexp: "^.*fix[(\\w)]*:+.*$"
+      order: 1
+    - title: Others
+      order: 999
 
-# https://goreleaser.com/customization/release/
+# Release configuration
 release:
-  draft: true
-  extra_files:
-    - glob: ./dist/*
-  header: |
-    # 🚀 Features
-    # 🎄 Enhancements
-    # 🐞 Notable bug fixes
-    # 🔖 Version updates
-    # 🎠 Community
\ No newline at end of file
+  # GoReleaser will auto-detect from git remote origin
+  mode: append
+  
+  # Create release draft
+  draft: false
```

Comment:
should always be a draft release, there should always be a manual review step on making releases public. for instance, the description will be pretty bad before manual curation